.github/workflows/ci.yml

```diff
@@ -7,36 +7,38 @@ on:
     branches: [ main ]
 
 jobs:
-  arm64_test:
-    name: CMake test arm64 (with modern protobuf,grpc and abseil)
-    runs-on: actuated-arm64-4cpu-16gb
-    steps:
-    - uses: actions/checkout@v4
-      with:
-        submodules: 'recursive'
-    - name: setup
-      env:
-        PROTOBUF_VERSION: '23.3'
-        ABSEIL_CPP_VERSION: '20230125.3'
-        CXX_STANDARD: '14'
-        CC: /usr/bin/gcc-10
-        CXX: /usr/bin/g++-10
-      run: |
-        sudo -E ./ci/setup_gcc10.sh
-        sudo -E ./ci/setup_cmake.sh
-        sudo -E ./ci/setup_ci_environment.sh
-        sudo -E ./ci/setup_googletest.sh
-        sudo -E ./ci/install_abseil.sh
-        sudo -E ./ci/install_protobuf.sh
-    - name: run otlp exporter tests
-      env:
-        CC: /usr/bin/gcc-10
-        CXX: /usr/bin/g++-10
-        WITH_ABSEIL: 'ON'
-        CXX_STANDARD: '14'
-      run: |
-        sudo -E ./ci/setup_grpc.sh -m -p protobuf -p abseil-cpp
-        ./ci/do_ci.sh cmake.exporter.otprotocol.test
+
+# Commented 2024-11-06, lack of workers in github causes CI failures
+#  arm64_test:
+#    name: CMake test arm64 (with modern protobuf,grpc and abseil)
+#    runs-on: actuated-arm64-4cpu-16gb
```

Comment:
We can also try using GitHub hosted runners - https://github.com/open-telemetry/community/blob/9d617b82cadd8940751f4223b3e2f0a99884d74c/assets.md#github-hosted-arm64-runners - as part of this or separate PR.

Reply:
Noted.

A separate PR is better, so we can unlock the builds and get CI flowing again now.